### PR TITLE
HBW-125 Small form improvements

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
image

**Comment by mxsmirnoff** [comment]
image

**Comment by mxsmirnoff** [comment]
image

File: hbw/app/assets/stylesheets/hbw.sass [hunk 1]
```diff
@@ -491,3 +489,10 @@ input.hbw-checkbox
   &[type=checkbox]
     + span:before
       content: "\f0c8"
+
+.react-select__control--is-disabled
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: I believe it works but have other way to customize react-select
https://github.com/latera/homs/blob/f680740201b65dccd09ae071edfb3e5501a9c4b7/hbw/app/javascript/packs/hbw/components/form/select.js.jsx#L125

**Comment by mxsmirnoff** [comment]
Updated

File: hbw/app/javascript/packs/hbw/components/form/select.js.jsx [hunk 2]
```diff
@@ -123,7 +123,15 @@ modulejs.define('HBWFormSelect',
       };
 
       customStyles = () => {
-        const bgColor = (state) => {
+        const bgControlColor = (state) => {
+          if (state.isDisabled) {
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/select.js.jsx]: Ternary?

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**